src/test/groovy/com/palantir/gradle/versions/VersionPropsIdeaPluginTest.groovy

```diff
@@ -0,0 +1,54 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.versions
+
+import nebula.test.IntegrationSpec
+
+class VersionPropsIdeaPluginTest extends IntegrationSpec {
+    def setup() {
```

Comment:
use [`@RestoreSystemProperties`](https://spockframework.org/spock/docs/2.3/extensions.html#_restoresystemproperties) for test hygiene

Reply:
I couldn't seem to apply this to the setup method but it did seem to work on the actual test - is that correct